data/example-1/dataset-metadata.tsv

```diff
@@ -1,13 +1,13 @@
 name	value
-citation	NA
-qiita-id	NA
+citation	hyperlink to original citation or pubmed listing (recommended), or optionally full citation details (not recommended). If not published, NA.
+qiita-id	Study ID if raw data are deposited in qiita. If not, list NA
 raw-data-url	http://www.qiime.org
-human-readable-description	This is an example data set for software testing and documentation purposes.
-bokulich2013-id	NA
-bokulich2015-id	NA
-target-gene	16S
-target-subfragment	V4
-study-type	marker-gene
-sequencing-instrument	illumina-miseq
-physical-specimen-available	No
-physical-specimen-contact	NA
+human-readable-description	Describe this dataset in as much detail as possible. Who generated these data, where, and when? How many strains and species are included in the mock community? How many samples? Were species mixed at equal ratios or uneven ratios? More is always better
+bokulich2013-id	NA. This field is only used for the original 11 datasets in mockrobiota, and lists the dataset ID listed in doi:10.1038/nmeth.2276. All new datasets should list NA for this field.
```

Comment:
I don't think I like having the descriptions in this file. It's intended to serve as a real example, and the documentation goes in ``CONTRIBUTING.md``. Could we remove this?